.github/workflows/backport-fail-bot.yml

```diff
@@ -0,0 +1,36 @@
+name: Forward failed backport alert to Slack
+
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  check_comment:
+    runs-on: ubuntu-latest
+    if: github.event.issue.pull_request != null && contains(github.event.comment.body, 'To backport manually, run these commands in your terminal')
+    steps:
+      - name: Generate Slack Payload
+        id: generate_payload
+        uses: actions/github-script@v4
+        with:
+          script: |
+            const slack_mapping = JSON.parse(process.env.SLACK_MAPPING);
+            const pr_url = "${{ github.event.issue.pull_request.html_url}}";
+            const pr_author_github_id = "${{ github.event.issue.user.login }}"
+            const pr_author_slack_id = slack_mapping[pr_author_github_id];
+            const author = (pr_author_slack_id ? `<@${pr_author_slack_id}>` : pr_author_github_id);
+            const payload = {
+              text: `Backport failed in PR: ${pr_url}. ${author} check it.`,
+              channel: process.env.SLACK_CHANNEL,
+            };
+            console.log(`::set-output name=payload::${JSON.stringify(payload)}`);
```

Comment:
https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/

The `::set-output` has been deprecated, could you please use the newer way?